ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -95,6 +95,13 @@ def pytest_addoption(parser):
         default=False,
         help="Run IO in the background",
     )
+    parser.addoption(
+        '--ocs-version',
+        dest='ocs_version',
+        action="store_true",
```

Comment:
why store_true? We should directly add the value of version and not boolean param if we passed the --ocs-version or not. Also not sure why default=False . There should be default version like 4.2 as I mentioned above in argparse.

Reply:
done